addons/osfstorage/models.py

```diff
@@ -517,6 +517,7 @@ def after_register(self, node, registration, user, save=True):
         clone = self.clone()
         clone.owner = registration
         clone.on_add()
+        clone.region_id = node.get_addon('osfstorage').region_id
```

Comment:
`after_fork` does this a few lines above:

```python
clone.region_id = self.region_id
```

Is there a reason you're doing it differently here?